modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/extra/remote-app/src/main.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {Root, createRoot} from 'react-dom/client';
+
+import './style/index.css';
+
+const HelloWorld = () => <div className="evp-app">Hello World</div>;
+
+type EVPComponentType = {
+	[key: string]: JSX.Element;
+};
+
+const EVPComponent: EVPComponentType = {
+	'hello-world': <HelloWorld />,
+};
+
+class EVPRemoteAppComponent extends HTMLElement {
+	public root: Root | undefined;
+
+	connectedCallback() {
+		type propertyType = {
+			route: string;
+		};
+
+		const properties: propertyType = {
+			route: this.getAttribute('route') || '',
```

Comment:
Add a fallback in case of route attribute is not present

route: this.getAttribute('route') || 'hello-world',